### PR TITLE
Fixed - Two <h1> on static frontpage

### PR DESCRIPTION
**Pull request by acosmin**
Fix for #490

### CODE REVIEW

**Comment by mukeshpanchal27** [comment]
@acosmin Above fix remove <h1> from Posts page.

**Comment by acosmin** [comment]
@mukeshpanchal27 it looks like that, I'll look into it... the same issue is available for twentynineteen

**Comment by acosmin** [comment]
@mukeshpanchal27 please go ahead and try it :) I think I got it right now 😄 I think a patch for twentynineteen is also needed

File: inc/template-tags.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ function twentytwenty_site_logo( $args = array(), $echo = true ) {
 			'title_class' => 'site-title',
 			'home_wrap'   => '<h1 class="%1$s">%2$s</h1>',
 			'single_wrap' => '<div class="%1$s faux-heading">%2$s</div>',
-			'condition'   => is_front_page() || is_home(),
+			'condition'   => ( is_front_page() && is_home() ) || ! is_singular(),
```
Comment (nielslange) [inline inc/template-tags.php]: @mukeshpanchal27 & @acosmin Currently, the following pages still show two `<h1>`:

- Author pages, e.g. http://wp.test/author/themedemos/
- Category pages, e.g. http://wp.test/category/markup/
- Tag pages, e.g. http://wp.test/tag/css/

I've tested this PR using [Screaming Frog SEO Spider](https://www.screamingfrog.co.uk/seo-spider/) which is available both as a free and a paid version. The free version should be more than sufficient for theme testing purposes. 😉

**Comment by acosmin** [comment]
I hope this one works :)

**Comment by nielslange** [comment]
> try nr.2

@acosmin As many folks contributing to this repo it would be better to use speaking commit messages such as `Enhance title conditions`, `Adjust <h1> conditions` or something along these lines instead of `try nr.2`, which could be pretty much everything. 😁

**Comment by acosmin** [comment]
twentynineteen also has some of these problems

**Comment by nielslange** [comment]
> twentynineteen also has some of these problems

You know how to fix it now, don't you? 😛

**Comment by acosmin** [comment]
I do but I hate trac :)

**Comment by pattonwebz** [comment]
I made a ticket XD https://core.trac.wordpress.org/ticket/48126

**Comment by nielslange** [comment]
> @acosmin This does not seem to fix the issue when using a static homepage. I still see 2 `<h1>`'s

Are you using the theme unit test file, @ianbelanger79. If so, there's a high chance that the second `<h1>` is coming from the post _Markup: HTML Tags and Formatting_, which contains `<h1>` until `<h6>`.

**Comment by acosmin** [comment]
@ianbelanger79 `is_page()` should take care of `is_page_template()` and other cases...

**Pull request merged**